tomls/omnibus-base-goerli-andromeda/spot/usdc.toml

```
@@ -5,9 +5,7 @@ defaultValue = "<%= parseEther(String(5_000_000)) %>"
 defaultValue = "60"
 
 [provision.usdc_mock_collateral]
-source = "mintable-token"
-targetPreset = "<%= settings.target_preset %>"
-options.salt = "<%= settings.salt %>"
+source = "mintable-token:1.3"
```

Need to use 1.7 and decimals 6
```suggestion
source = "mintable-token:1.7"
options.decimals = "6"
```

---

This was deployed as is though? I don't think we should change it if that's the case.

---

We need to have fUSDC to have same decimals as real usdc because all the tests and potentially integrators implementations assume 18 decimals. Which will be a problem when we deploy to mainnet. We need to have a testnet deploy that is close to mainnet to avoid such issues